test/go-chaincode/build.bats

```diff
@@ -1,5 +1,31 @@
 #!/usr/bin/env bats
 
-@test "Placeholder golang build test script" {
+setup() {
+  src_dir="scripts"
+}
+
+@test "build.sh: should exist and be executable" {
+  [ -x "${src_dir}/go-chaincode/build.sh" ]
+}
+
+@test "build.sh: should install fabric STUB" {
```

Comment:
There should be a go-chaincode/download-fabric.bats file to test the go install, rather than putting tests in here. (Worth stubbing curl and possibly tar as well for that.)